pkg/sql/logictest/testdata/logic_test/geospatial

```diff
@@ -1013,161 +1013,162 @@ SELECT ST_IsRing('POINT(0 0)')
 
 # Topology operators
 
-query TTTBT
+query TTTBTT
 SELECT
   a.dsc,
   b.dsc,
   ST_AsEWKT(ST_Intersection(a.geom, b.geom)),
   ST_Intersection(a.geom, b.geom) = ST_Intersection(a.geom::string, b.geom::string),
-  ST_AsEWKT(ST_SymDifference(a.geom, b.geom))
+  ST_AsEWKT(ST_SymDifference(a.geom, b.geom)),
+  ST_AsEWKT(ST_SymmetricDifference(a.geom, b.geom))
```

Comment:
thanks! quick comment -- can you move both these symdifference statements to a separate select?

Reply:
Done 👍